packages/nvidia_nat_nemo_customizer/tests/test_nemo_customizer.py

```
@@ -0,0 +1,1118 @@
+        mock_adapter.initialize = AsyncMock()
+
+        await trainer.bind_components(mock_builder, mock_adapter)
+        await trainer.initialize(finetune_config)
+
+        statuses = await trainer.run(num_epochs=3)
+
+        assert len(statuses) == 1
+        assert statuses[0].status == TrainingStatusEnum.FAILED
+        assert "No trajectories collected" in statuses[0].message
+
+    def test_deduplicate_trajectories(self, trainer, sample_dpo_trajectories):
```

Ruff (0.14.8)

Unused method argument: `sample_dpo_trajectories`

(ARG002)